### ISSUE TITLE
Labels on y-axis do not match plotted values in scalars view

### ISSUE THREAD

**Issue by Dobiasd** [tensorflow/tensorboard#1753]
Just stumbled over something similar as [this old issue](https://github.com/tensorflow/tensorflow/issues/9056). So I created a minimal example to reproduce the effect:

```python
# tensorboard_y_axis_labels_issue.py

import keras
import numpy as np
from keras.layers import Input, Dense
from keras.models import Model

inputs = Input(shape=(100,))
x = Dense(80, activation='tanh')(inputs)
predictions = Dense(10, activation='softmax')(x)
model = Model(inputs=inputs, outputs=predictions)
model.compile(loss='categorical_crossentropy', optimizer='nadam')

x_train = np.random.rand(10000, 100)
y_train = np.random.rand(10000, 10)

model.fit(
    x=x_train,
    y=y_train,
    epochs=1000,
    callbacks=[keras.callbacks.TensorBoard(log_dir='tensorboard_logs')])
```

Screenshot of effect:

![tensorboard_screenshot](https://i.imgur.com/gNVD8EP.png)

From the labels of the y-axis I'd expect the highlighted point to be `> 11.5` instead of `11.48`.

Command line:
```bash
python3 tensorboard_y_axis_labels_issue.py
tensorboard --logdir tensorboard_logs
```

Used versions:

```bash
$ pip3 list | grep tensor
tensorboard              1.12.2
tensorflow               1.12.0

$ inxi -S
System:    Host: tobias-Z170-HD3P Kernel: 4.13.0-38-generic x86_64 (64 bit) Desktop: MATE 1.18.0
           Distro: Linux Mint 18.2 Sonya

$ python3 --version
Python 3.6.8
```

**Comment by wchargin** [comment]
Can reproduce with

```
$ pip list | grep Keras
Keras               2.2.4      
Keras-Applications  1.0.6      
Keras-Preprocessing 1.0.5      
```

(which is latest `pip install keras`).

Can reproduce at various levels of smoothing (0, 0.6, 0.8).

**Comment by wchargin** [comment]
Event files for self-contained repro attached. Mouse over step 28, e.g.

[repro_events.zip](https://github.com/tensorflow/tensorboard/files/2751082/repro_events.zip)
